### PR TITLE
UG small changes

### PR DESCRIPTION
**Pull request by sean-phang**
A few minor edits here and there, please look through, my english is not the best.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -46,19 +46,20 @@ This guide serves to help you understand the usage of the program, which include
...
-| Recipe                      | Acts as a container for you to link several modules together that will be executed back-to-back on your data. By default, the application does not have any default recipes installed. |
-| Data transformtions         | Refers to any process that converts data from one format to another which includes encoding, decoding, encryption and decryption.                            | 
+| Terminology                 | Definition                                                                                                                                                                                |
+|-----------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Data transformation         | The conversion of one data format to another.                                                                                                                                             |
```
Comment (arraysius) [inline docs/UserGuide.md:51]: Duplicate definition for data transformation

File: docs/UserGuide.md [hunk 2]
```diff
@@ -120,7 +121,7 @@ Below, you can see the list of available modules and what they do. <a name="sele
 
 #### Help: `help`
 
-Lists all syntaxes and descriptions of available commands.
+Lists the syntax(s) and descriptions of all available commands.
```
Comment (arraysius) [inline docs/UserGuide.md:124]: I think the "(s)" can be omitted and just left as "syntax"

File: docs/UserGuide.md [hunk 3]
```diff
@@ -331,7 +332,7 @@ Coming soon to a cinema near you in the future…
 
 | Action                                      | Command Format                                                             | Example Usage                                                                             |
 | ------------------------------------------- | -------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- |
-| View syntaxes and description of commands   | `help`                                                                     | `help`                                                                                    |
+| View the syntax(s) and description of commands | `help`                                                                     | `help`                                                                                    |
```
Comment (arraysius) [inline docs/UserGuide.md:335]: "(s)" can probably be left out.

**Review by arraysius — APPROVED** [review]
Some minor change requests but good overall

**Review by Kair0s3 — APPROVED** [review]
Other than those pointed out, looks good!

**Pull request merged**